component.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/moqui-conf-3.xsd"
+           name="mantle-shipengine" version="1.2.0">
+    <depends-on name="mantle-usl" version="2.2.0"/>
```

Comment:
I don't see any OOTB service being used in this code so we shouldn't be needing this dependency. In fact we are using OOTB data model, so dependency should be added for mantle-udm instead. Or may be just leave a comment.